src/main/scala/mesosphere/marathon/state/LinuxInfo.scala

```diff
@@ -0,0 +1,31 @@
+package mesosphere.marathon
+package state
+
+/**
+  * Defines the seccomp expectations for the instance this is associated with.   This will govern the secure computing mode.
+  * Unlike Mesos, unconfined is not optional.  If seccomp is provided, then unconfined is either true or false.  The optionality is with seccomp itself.
+  *
+  * @param profileName The profile name which defines the security model this container will run under.  It is required that this profile be defined at the agent.
+  * @param unconfined True is running under a profile, False if under a profile.
```

Comment:
I think you mean `True is not running under a profile, False if under a profile`?